keyboards/ocean/gin_v2/readme.md

```diff
@@ -0,0 +1,24 @@
+# Gin V2
+
+![Gin V2](https://media.discordapp.net/attachments/406032987243806721/862564309406056468/20210708_131916.jpg?width=1166&height=657)
```

Comment:
It would be better to have this posted on a dedicated image site, such as imgur.

Reply:
changed to imgur, thank you for the suggestion.